rpcs3/rpcs3qt/save_data_list_dialog.cpp

```diff
@@ -196,7 +134,7 @@ s32 save_data_list_dialog::GetSelection()
 
 void save_data_list_dialog::OnSort(int idx)
 {
-	if ((idx < m_sort_type_count) && (idx >= 0))
```

Comment:
It wasn't needed so I removed it.  In fact, not changing it from 3 to 4 broke sorting for entry notes. It annoyed me enough I deleted it for now.  Probably would be useful if we wanted to add sorting by other stats but for now, if we're only sorting by columns, it makes code simpler to leave it as is.